miq_server_leak_discovery/simple_memory_monitor.rb

```diff
@@ -1,6 +1,31 @@
 require 'sys-proctable'
 
+# coma separated values - easier to read
+def delaminate(number)
```

Comment:
While I think "delaminate" technically is right (based on is definition), would "delimitate" seems like the more proper word (derived from "delimiter").

Reply:
Or maybe it is "delimit"... english isn't my thing...